core/tests/build_sources/assets/constants.js

```diff
@@ -10,6 +10,7 @@
 var constants = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
+  "CAN_SEND_EMAILS": false,
```

Comment:
Add the same comment as in `assets/constants.js`

Reply:
@vojtechjelinek Done Please review and merge